src/app/app.module.ts

```diff
@@ -13,7 +13,7 @@ import { ProductListComponent } from './product-list/product-list.component';
     ReactiveFormsModule,
     RouterModule.forRoot([
       { path: '', component: ProductListComponent },
-    ])
+    ], { useHash: true })
```

Comment:
We should add a TypeScript comment before this line that says something like:

STACKBLITZ WILL REMOVE THE `{ useHash: true }` ENTRY, BUT THIS IS CRITICAL FOR THE GITHUB PAGES DEPLOYMENT TO WORK. MAKE SURE YOU PRESERVE THIS ACROSS UPDATES!